src/styles/classes.module.css

```diff
@@ -0,0 +1,7 @@
+.disableSelect {
+  user-select: none; /* supported by Chrome and Opera */
+ -webkit-user-select: none; /* Safari */
```

Comment:
`classes.module.css` is a global stylesheet for common, usually single attribute, styles we will probably need for things in the future. We are using this to avoid too much repetition